### ISSUE TITLE
When I restart an application in "suspend" debug mode I'm not shown debug connection info

### ISSUE THREAD

**Issue by youngm** [cloudfoundry-attic/cf#47]
This is a reopen of issue https://github.com/cloudfoundry/cf/issues/35  I'd like to see this issue flagged as "open for contribution".  I may contribute a fix someday. :)

**Comment by tlang** [comment]
Yep, that is a valid request - this issue is "open for contribution". Stand by for the label. And thanks for your suggestion and contribution!

**Comment by youngm** [comment]
Are the issues "Open for Contribution" supposed to be closed?

**Comment by tlang** [comment]
Sorry about that youngm. We had originally drafted a policy of closing this category of issue, but upon further reflection, yes you're right it's confusing and we'll leave them open.

**Comment by youngm** [comment]
No problem.  It was an honest question.  I think I like them being open as well.

Mike
